config/custom.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+    'url' => [
+    'prefix_admin' => 'admin',
+    ]
+];
```

Comment:
```suggestion
return [
    'url' => [
        'prefix_admin' => 'admin',
    ]
];
```
thiếu dòng trắng cuối file